docs/development/models-migrations.md

```diff
@@ -9,7 +9,7 @@
 Cal-ITP Benefits defines a number of [models][core-models] in the core application, used throughout the codebase to configure
 different parts of the UI and logic.
 
-The Cal-ITP Benefits database is a simple read-only Sqlite database, initialized from the [fixture configuration](../configuration/fixtures.md) files.
+The Cal-ITP Benefits database is a simple read-only Sqlite database, initialized from the [data migration](../configuration/data.md) files.
 
 ## Migrations
```

Comment:
Have we confirmed what happens here?

E.g. make a change to the schema in `0001` migration, run `bin/makemigrations.sh` - does this affect the `002` (and `003` from #827) migrations? Do we need to manually change those?

Reply:
Tried this out and then made changes in https://github.com/cal-itp/benefits/pull/814/commits/3311a8a10c3fd66bec94a4b3850b1fe77dd92c61

Reply:
(Not necessarily a bad thing) but this seems to be working differently now.

I just made a simple change to the `PemData` model and ran `bin/makemigrations.sh` -- the result was a new `003_alter_pem_data.py` file that had the change, `001` was unaffected.

Perhaps now is the time to forgo the "one schema migration to rule them all" concept, since we now do have multiple migrations to deal with.